src/main/resources/log4j.properties

```diff
@@ -16,8 +16,16 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-log4j.rootLogger=INFO, stdout
-log4j.appender.stdout=org.apache.log4j.ConsoleAppender
-log4j.appender.stdout.Target=System.out
-log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
-log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
\ No newline at end of file
+# Set root logger level to DEBUG.
+log4j.rootLogger=DEBUG, console, myFile
+
+log4j.appender.console=org.apache.log4j.ConsoleAppender
+log4j.appender.console.layout=org.apache.log4j.PatternLayout
+log4j.appender.console.layout.ConversionPattern=[%d{ABSOLUTE} %-5p %c{1}]: %m%n
+
+# myFile writes to file
+log4j.appender.myFile=org.apache.log4j.RollingFileAppender
```

Comment:
what are these for?

Reply:
Trying to get some debugging output. Reverted since it didn't work anyway.